src/main/webapp/app/admin/health/health.component.html

```diff
@@ -16,20 +17,18 @@ <h2>
             <tbody *ngIf="health">
                 <tr *ngFor="let componentHealth of health.components | keyvalue">
                     <td>
-                        <span>
-                            {{ 'health.indicator.' + componentHealth.key | artemisTranslate }}
-                            <a *ngIf="componentHealth.value?.details?.url" class="badge bg-info" [href]="componentHealth.value?.details.url">
-                                {{ componentHealth.value?.details.url }}
-                            </a>
-                        </span>
+                        {{ 'health.indicator.' + componentHealth.key | artemisTranslate }}
+                        <a *ngIf="componentHealth.value?.details?.url" class="badge bg-info" [href]="componentHealth.value!.details!.url">
```

Comment:
```suggestion
                        <a *ngIf="componentHealth.value!.details?.url" class="badge bg-info" [href]="componentHealth.value!.details!.url">
```
This should not be undefined. If it would be, we would have issues with the usage below.

Reply:
Not necessary since generated from jhipster